### ISSUE TITLE
Provide example outputs for validation

### ISSUE THREAD

**Issue by monocongo** [monocongo/climate_indices#223]
Provide example outputs (including both NetCDF datasets and associated plot images) corresponding to the example inputs, for validation/clarification, etc.

**Comment by monocongo** [comment]
Output datasets created using the new `process_grid_map.py` script (currently from the develop branch, but that code will soon replace the `process_grid.py` script):

```
$ cd climate_indices
$ python scripts/process_grid_map.py \
--index all \
--periodicity monthly \
--scales 1 2 3 6 9 12 24 \
--netcdf_precip example/input/nclimgrid_lowres_prcp.nc \
--var_name_precip prcp \
--netcdf_temp example/input/nclimgrid_lowres_tavg.nc \
--var_name_temp tavg \
--netcdf_awc example/input/nclimgrid_lowres_soil.nc \
--var_name_awc awc \
--output_file_base example/output/nclimgrid_lowres \
--calibration_start_year 1951 \
--calibration_end_year 2010 \
--multiprocessing all

2018-12-14  14:30:56 INFO Start time:    2018-12-14 14:30:56.183190
2018-12-14  14:30:56 INFO Computing 1-month SPI/Pearson
2018-12-14  14:31:04 INFO Computing 1-month SPI/Gamma
2018-12-14  14:31:08 INFO Computing 2-month SPI/Pearson
2018-12-14  14:31:16 INFO Computing 2-month SPI/Gamma
2018-12-14  14:31:20 INFO Computing 3-month SPI/Pearson
2018-12-14  14:31:28 INFO Computing 3-month SPI/Gamma
2018-12-14  14:31:32 INFO Computing 6-month SPI/Pearson
2018-12-14  14:31:40 INFO Computing 6-month SPI/Gamma
2018-12-14  14:31:45 INFO Computing 9-month SPI/Pearson
2018-12-14  14:31:53 INFO Computing 9-month SPI/Gamma
2018-12-14  14:31:58 INFO Computing 12-month SPI/Pearson
2018-12-14  14:32:06 INFO Computing 12-month SPI/Gamma
2018-12-14  14:32:11 INFO Computing 24-month SPI/Pearson
2018-12-14  14:32:20 INFO Computing 24-month SPI/Gamma
2018-12-14  14:32:24 INFO Computing PET
2018-12-14  14:32:36 INFO Computing 1-month SPEI/Pearson
2018-12-14  14:32:45 INFO Computing 1-month SPEI/Gamma
2018-12-14  14:32:50 INFO Computing 2-month SPEI/Pearson
2018-12-14  14:32:59 INFO Computing 2-month SPEI/Gamma
2018-12-14  14:33:04 INFO Computing 3-month SPEI/Pearson
2018-12-14  14:33:13 INFO Computing 3-month SPEI/Gamma
2018-12-14  14:33:17 INFO Computing 6-month SPEI/Pearson
2018-12-14  14:33:25 INFO Computing 6-month SPEI/Gamma
2018-12-14  14:33:30 INFO Computing 9-month SPEI/Pearson
2018-12-14  14:33:39 INFO Computing 9-month SPEI/Gamma
2018-12-14  14:33:43 INFO Computing 12-month SPEI/Pearson
2018-12-14  14:33:51 INFO Computing 12-month SPEI/Gamma
2018-12-14  14:33:55 INFO Computing 24-month SPEI/Pearson
2018-12-14  14:34:03 INFO Computing 24-month SPEI/Gamma
2018-12-14  14:34:07 INFO Computing 1-month PNP
2018-12-14  14:34:09 INFO Computing 2-month PNP
2018-12-14  14:34:11 INFO Computing 3-month PNP
2018-12-14  14:34:13 INFO Computing 6-month PNP
2018-12-14  14:34:15 INFO Computing 9-month PNP
2018-12-14  14:34:17 INFO Computing 12-month PNP
2018-12-14  14:34:19 INFO Computing 24-month PNP
2018-12-14  14:34:21 INFO Computing PALMERS
2018-12-14  14:42:36 INFO End time:      2018-12-14 14:42:36.494707
2018-12-14  14:42:36 INFO Elapsed time:  0:11:40.311517

Process finished with exit code 0
```

**Comment by monocongo** [comment]
We will also attempt to utilize [git-lfs](https://git-lfs.github.com/) for managing these files.

**Comment by monocongo** [comment]
As a result of having so many data files in the project, it now takes quite some time to clone, even after halving the size of the output files as a result of work done for #227. This causes Travis builds to take about twice as long. We don't want unecessary data transfer for each Travis build nor do we want users to have to pull so much data when it's probably not necessary in most cases.

Next steps:

- [x] Move the example input/output files into a separate/auxiliary repository
- [x] Update the documentation accordingly

**Comment by monocongo** [comment]
Example input and output NetCDF dataset files and associated plot images are now available at [example_climate_indices](https://github.com/monocongo/example_climate_indices).

This issue resolved by PR #229
